Sources/Routes/FourOhFour.swift

```diff
@@ -6,10 +6,32 @@ public class FourOhFour {
     public init(){}
 
     public func handleRoute(request: Request) -> String {
+        let filePath = NSURL.fileURL(withPathComponents: ["requestLog.txt"])
+        let logData = readLogRequest(path: filePath!)
```

Comment:
You seem to be reading the entire contents of `requestLog.txt` only to write it again. Why? What problem are you solving?

Reply:
I can address this issue by finding a means to append text as opposed to rewriting over text.

Reply:
There should be a way in Swift's I/O APIs to append.